libflux/src/flux/semantic/mod.rs

```diff
@@ -29,15 +29,21 @@ use crate::semantic::analyze::Result as AnalysisResult;
 use crate::semantic::analyze::SemanticError;
 use crate::semantic::env::Environment;
 use crate::semantic::fresh::Fresher;
+use crate::semantic::import::Importer;
 use crate::semantic::nodes::Error as InferError;
-use crate::semantic::nodes::{infer_pkg_types, inject_pkg_types, Importer};
+use crate::semantic::nodes::{infer_pkg_types, inject_pkg_types};
+use crate::semantic::types::PolyType;
 
 impl From<InferError> for SemanticError {
     fn from(err: InferError) -> SemanticError {
         err.msg.clone()
     }
 }
 
+struct TypeImporter {}
+
+impl Importer<PolyType> for TypeImporter {}
```

Comment:
Can you unpack this for me?  I don't follow how you can say `TypeImporter` implements `Importer` if it doesn't define an `import` method.

Reply:
Oh, never mind, I just figured out it has a default implementation that always returns `None`.

Reply:
So it's this type that will eventually be replaced with the importer for the stdlib, right?

Reply:
Yes that's correct.